sources/packages/api/src/services/config/config.service.ts

```diff
@@ -56,6 +56,13 @@ export class ConfigService {
         programAreaCode: process.env.CRA_PROGRAM_AREA_CODE,
         environmentCode: process.env.CRA_ENVIRONMENT_CODE,
       },
+      ATBCIntegration: {
+        ATBCloginEndpoint: process.env.ATBC_LOGIN_ENDPOINT,
```

Comment:
Please keep the same pattern as the others in the same object: "ATBCLoginEndpoint"